docs/sources/analyzing-profiles-in-pyroscope-ui/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "How to use the Pyroscope UI for performance analysis"
+menuTitle: "How to use the Pyroscope UI"
+description: "How to use the Pyroscope UI to analyze performance of your applications."
+weight: 30
+keywords:
+  - pyroscope
+  - UI
+  - performance analysis
+  - flamegraphs
+---
+
+# How to use the Pyroscope UI for performance analysis
+
+## Introduction
+
+**Understanding Continuous Profiling and Metadata in Pyroscope**
+
+While code profiling has been a long-standing practice, Continuous Profiling represents a modern and more advanced approach to performance monitoring. This technique adds two critical dimensions to traditional profiles:
+
+- **Time:** Profiling data is collected _continuously_, providing a time-centric view that allows querying performance data from any point in the past
+- **Metadata:** Profiles are enriched with metadata, adding contextual depth to the performance data
+
+These dimensions, coupled with the detailed nature of performance profiles, make Continuous Profiling a uniquely valuable tool. Pyroscope's UI enhances this further by offering a convenient platform to analyze profiles and get insights that are impossible to get from using other traditional signals like logs, metrics, or tracing. 
+
+In this walkthrough, we'll show how Pyroscope parallels these other modern observability tools by providing a Prometheus-like querying experience. More importantly, you'll learn how to use Pyroscope's extensive UI features for a deeper insight into your application's performance.
```

Comment:
This entire section has great information, but it says that "in this walkthrough" -- except what follows is a UI reference page and a Pyroscope in Grafana page. Does this content really belong as an introduction ot the Why use continuous profiling? page? 

What would you think about breaking this page into multiple sections? It feels like this page is trying to do several things. It might be easier for a user to understand as separate pages. 

1. Pyroscope UI overview (see examples, [Grafana dashboards overview](https://grafana.com/docs/grafana/latest/fundamentals/dashboards-overview/) and [Panel editor overview](https://grafana.com/docs/grafana/latest/panels-visualizations/panel-editor-overview/) 
2.  Pyroscope in Grafana - as separate page that provides an example of what Pyroscope in Grafana provides  (see example, [Tempo in Grafana](https://grafana.com/docs/tempo/latest/getting-started/tempo-in-grafana/)  
3. Continuous profiling content - either move to it's own page or add to why use continuous profilng page as an intro

Reply:
I see yeah I'll rephrase it to say UI reference. I will create an issue for this as I think I'd need to see this to see what it would look like split up